[2025-05-08T09:33:38.597+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: lgbtq_model_pipeline.human_review manual__2025-05-08T09:33:29.499705+00:00 [queued]>
[2025-05-08T09:33:38.602+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: lgbtq_model_pipeline.human_review manual__2025-05-08T09:33:29.499705+00:00 [queued]>
[2025-05-08T09:33:38.602+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2025-05-08T09:33:38.608+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): human_review> on 2025-05-08 09:33:29.499705+00:00
[2025-05-08T09:33:38.624+0000] {standard_task_runner.py:57} INFO - Started process 365 to run task
[2025-05-08T09:33:38.632+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'lgbtq_model_pipeline', 'human_review', 'manual__2025-05-08T09:33:29.499705+00:00', '--job-id', '274', '--raw', '--subdir', 'DAGS_FOLDER/lgbtq_model_pipeline.py', '--cfg-path', '/tmp/tmp16ocso7s']
[2025-05-08T09:33:38.642+0000] {standard_task_runner.py:85} INFO - Job 274: Subtask human_review
[2025-05-08T09:33:38.668+0000] {task_command.py:416} INFO - Running <TaskInstance: lgbtq_model_pipeline.human_review manual__2025-05-08T09:33:29.499705+00:00 [running]> on host 6deebfaa31ce
[2025-05-08T09:33:38.715+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='lgbtq_model_pipeline' AIRFLOW_CTX_TASK_ID='human_review' AIRFLOW_CTX_EXECUTION_DATE='2025-05-08T09:33:29.499705+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-08T09:33:29.499705+00:00'
[2025-05-08T09:33:38.716+0000] {logging_mixin.py:154} INFO - ðŸ‘€ Checking for enough articles in review queue...
[2025-05-08T09:33:38.722+0000] {logging_mixin.py:154} WARNING - /home/***/.local/lib/python3.8/site-packages/pymongo/synchronous/mongo_client.py:2554 UserWarning: MongoClient opened before fork. May not be entirely fork-safe, proceed with caution. See PyMongo's documentation for details: https://www.mongodb.com/docs/languages/python/pymongo-driver/current/faq/#is-pymongo-fork-safe-
[2025-05-08T09:33:39.404+0000] {logging_mixin.py:154} INFO - âœ… Found 173 files â€” simulating human review...
[2025-05-08T09:33:39.405+0000] {logging_mixin.py:154} INFO - ðŸ’Œ Sending email to team...
[2025-05-08T09:33:39.406+0000] {logging_mixin.py:154} INFO - hiyyyy time 2 label these new entries babe okthxbye
[2025-05-08T09:33:39.941+0000] {logging_mixin.py:154} INFO - ðŸ“¤ Reviewed + moved article 681c71b308ab8217dc5f7a39 â†’ new_data/
[2025-05-08T09:33:40.333+0000] {logging_mixin.py:154} INFO - ðŸ“¤ Reviewed + moved article 681c71b408ab8217dc5f7a3a â†’ new_data/
[2025-05-08T09:33:40.698+0000] {logging_mixin.py:154} INFO - ðŸ“¤ Reviewed + moved article 681c71b408ab8217dc5f7a3b â†’ new_data/
[2025-05-08T09:33:41.058+0000] {logging_mixin.py:154} INFO - ðŸ“¤ Reviewed + moved article 681c71b508ab8217dc5f7a3c â†’ new_data/
[2025-05-08T09:33:41.435+0000] {logging_mixin.py:154} INFO - ðŸ“¤ Reviewed + moved article 681c71b608ab8217dc5f7a3d â†’ new_data/
[2025-05-08T09:33:41.436+0000] {logging_mixin.py:154} INFO - ðŸ§  Human review simulated for this batch.
[2025-05-08T09:33:41.436+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-05-08T09:33:41.464+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=lgbtq_model_pipeline, task_id=human_review, execution_date=20250508T093329, start_date=20250508T093338, end_date=20250508T093341
[2025-05-08T09:33:41.618+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-05-08T09:33:41.659+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
